packages/manager/modules/vps/src/dashboard/tile/configuration/upgrade/component.js

```diff
@@ -0,0 +1,11 @@
+import controller from './controller';
+import template from './index.html';
+
+export default {
+  name: 'vpsDashboardTileConfigurationUpgrade',
+  controller,
+  template,
+  bindings: {
+    resolve: '<',
```

Comment:
It sounds confusing to have a binding call `resolve`  with ui router. What is its purpose ?

Reply:
This is because the component is used by a `oui-modal` layout state. To be compatible with new version of `uibModal` resolves are accessible through resolve binding of the component. See https://github.com/ovh/manager/blob/feat/vps-new-range-configuration/packages/components/ng-ui-router-layout/src/oui-modal/index.js#L61